client/webserver/site/src/js/markets.js

```diff
@@ -401,6 +401,15 @@ export default class MarketsPage extends BasePage {
     const market = this.market
     const page = this.page
     const [b, q] = [market.base, market.quote]
+    if (!b || !q) {
+      const getBaseSymbol = (m) => market.baseCfg.symbol
+      const getQuoteSymbol = (m) => market.quoteCfg.symbol
+      const unkownAssetName = !b ? getBaseSymbol(b) : getQuoteSymbol(q)
+      Doc.show(page.loaderErr)
+      page.loaderErr.textContent = `Market Asset "${unkownAssetName}" is not known`
+      return
+    }
+    Doc.hide(page.loaderErr)
     if (data.base !== b.id || data.quote !== q.id) return
```

Comment:
I'd like to still display the chart, if possible. I just realized that we might be able to get the missing asset info from `app.user.exchanges[url].assets` instead. I wonder if we can make that work.